package.json

```diff
@@ -5,13 +5,13 @@
   "main": "lib/index.js",
   "scripts": {
     "prepublish": "not-in-publish || safe-publish-latest && npm run build",
-    "build": "rimraf lib && babel src -d lib",
+    "build": "rimraf lib && babel --source-maps inline src -d lib",
```

Comment:
Hmm, running into some problems here.

The higher quality source maps decreases the coverage which isn't a big deal (I guess this was the point :D), but I can't get nyc to map back to original sources.

I tried both with and without the `babel-plugin-istanbul` plugin, but I couldn't get the mapping back from the instrumentation code+es5/es5 respectively.

I can't find anything in the `nyc` or `babel-plugin-istanbul` docs about external source maps - I also notice the documentation seems to recommend using `inline` source maps (https://github.com/istanbuljs/nyc#support-for-custom-require-hooks-babel-webpack-etc)

I'll open an issue+reproducible example on nyc later, but for now, this is blocking me :(

Have you got any thoughts w/r solving this? (You mentioned you had another repo set up with nyc?)